tests/types/double.spec.ts

```diff
@@ -0,0 +1,102 @@
+import {StrongDouble, makeDouble} from '../../src/types/double';
+
+const MOCK_INITIAL = 5246576;
+const MOCK_FALLBACK_DEFAULT = 6.5;
+const MOCK_FALLBACK = -9.596596;
+
+describe('StrongDouble', () => {
+	describe('Implementation', () => {
+		it('should set value to initialValue argument', () => {
+			const sampleInitial = 5.785216;
+			const result = makeDouble(sampleInitial, MOCK_FALLBACK_DEFAULT);
+			expect(result()).toBe(sampleInitial);
+		});
+
+		it('should return fallback default when value is null', () => {
+			const sampleFallback = 999981;
+			const result = makeDouble(null, sampleFallback);
+			expect(result()).toBe(sampleFallback);
+		});
+
+		it('should return fallback default when value is undefined', () => {
+			const sampleFallback = 5.664546634;
+			const result = makeDouble(undefined, sampleFallback);
+			expect(result()).toBe(sampleFallback);
+		});
+
+		it('should set value when called with non-null value', () => {
+			const sampleValue = 14.81;
+			const result = makeDouble(null, MOCK_INITIAL);
+			result(sampleValue);
+			expect(result()).toBe(sampleValue);
+		});
+
```

Comment:
Add a test for calling with `0`. Most tests involving numbers should test `0` specifically because it is the only number that coerces to false.